common/api/core-geometry.api.md

```diff
@@ -3860,12 +3860,12 @@ export class Order5Bezier extends BezierCoffs {
     sumBasisFunctions(u: number, polygon: Float64Array, n: number, result?: Float64Array): Float64Array;
 }
 
-// @alpha
+// @public
```

Comment:
Since we plan to refactor `OrderedRotationAngles`, let's keep it @alpha instead of promoting it to @public. One advantage to this is that it's OK to make the (potentially) breaking change of adding undefined to the return type of `createFromMatrix3d`.

Reply:
sure!